api/codacy.ts

```diff
@@ -34,40 +33,39 @@ const SUBJECT_BY_TYPE = {
   grade: 'code quality'
 }
 
-async function handler ({ type, projectId, branch = 'master' }: PathArgs) {
-  if (projectId) {
-    const svg = await got(`${uriBase}/${type}/${projectId}`,{
-      searchParams: { branch: branch && branch.replace('-', '--') }
```

Comment:
@amio I have no idea where did this come from? :confused: Reading through [docs](https://docs.codacy.com/repositories/badges/) I haven't seen mention of branch name _escaping_ :confused:

Reply:
I don't know either = =

Guess we need to find another repo to demonstrate this feature
Found this one https://github.com/codacy/codacy-coverage-reporter (1c524e61cd8640e79b80d406eda8754b)

Reply:
@amio We won't get far with that one too :tired_face: 

- default branch coverage: ![](https://api.codacy.com/project/badge/coverage/1c524e61cd8640e79b80d406eda8754b)
- default branch grade: ![](https://api.codacy.com/project/badge/grade/1c524e61cd8640e79b80d406eda8754b)
- [`script-tests`](https://github.com/codacy/codacy-coverage-reporter/tree/script-tests) branch coverage: ![](https://api.codacy.com/project/badge/coverage/1c524e61cd8640e79b80d406eda8754b?branch=script-tests)
- [`script-tests`](https://github.com/codacy/codacy-coverage-reporter/tree/script-tests) branch grade: ![](https://api.codacy.com/project/badge/grade/1c524e61cd8640e79b80d406eda8754b?branch=script-tests)

<details>
<summary>View source</summary>

```md
- default branch coverage: ![](https://api.codacy.com/project/badge/coverage/1c524e61cd8640e79b80d406eda8754b)
- default branch grade: ![](https://api.codacy.com/project/badge/grade/1c524e61cd8640e79b80d406eda8754b)
- [`script-tests`](https://github.com/codacy/codacy-coverage-reporter/tree/script-tests) branch coverage: ![](https://api.codacy.com/project/badge/coverage/1c524e61cd8640e79b80d406eda8754b?branch=script-tests)
- [`script-tests`](https://github.com/codacy/codacy-coverage-reporter/tree/script-tests) branch grade: ![](https://api.codacy.com/project/badge/grade/1c524e61cd8640e79b80d406eda8754b?branch=script-tests)
```
</details>

Reply:
@amio I figured out which project was originally used: https://github.com/xobotyi/react-scrollbars-custom but branch badges still don't work :shrug: 

- default branch coverage: ![](https://api.codacy.com/project/badge/coverage/f0875490cea1497a9eca9c25f3f7774e)
- default branch grade: ![](https://api.codacy.com/project/badge/grade/f0875490cea1497a9eca9c25f3f7774e)
- [`dev-master`](https://github.com/xobotyi/react-scrollbars-custom/tree/dev-master) branch coverage: ![](https://api.codacy.com/project/badge/coverage/f0875490cea1497a9eca9c25f3f7774e?branch=dev-master)
- [`dev-master`](https://github.com/xobotyi/react-scrollbars-custom/tree/dev-master) branch grade: ![](https://api.codacy.com/project/badge/grade/f0875490cea1497a9eca9c25f3f7774e?branch=dev-master)

<details>
<summary>View source</summary>

```md
- default branch coverage: ![](https://api.codacy.com/project/badge/coverage/f0875490cea1497a9eca9c25f3f7774e)
- default branch grade: ![](https://api.codacy.com/project/badge/grade/f0875490cea1497a9eca9c25f3f7774e)
- [`dev-master`](https://github.com/xobotyi/react-scrollbars-custom/tree/dev-master) branch coverage: ![](https://api.codacy.com/project/badge/coverage/f0875490cea1497a9eca9c25f3f7774e?branch=dev-master)
- [`dev-master`](https://github.com/xobotyi/react-scrollbars-custom/tree/dev-master) branch grade: ![](https://api.codacy.com/project/badge/grade/f0875490cea1497a9eca9c25f3f7774e?branch=dev-master)
```
</details>

Reply:
I guess we have to get rid of the branch example for now  =.=

Reply:
Or we could kinda cheat by using `?branch=master` as shields examples do 🤷

Reply:
That's not bad 😂

Reply:
Done :heavy_check_mark: